ts/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -583,6 +583,14 @@ class LocalTrainingService implements TrainingService {
         const filepath: string = path.join(directory, generateParamFileName(hyperParameters));
         await fs.promises.writeFile(filepath, hyperParameters.value, { encoding: 'utf8' });
     }
+
+    public getTrialOutputLocalPath(_trialJobId: string): Promise<string> {
+        throw new MethodNotImplementedError();
```

Comment:
Tensorboard does not support local for now?

Reply:
Yes, because webUI won't support tensorboard in this rls, I will support `non-reuse local` in next pr. Now, it supports `reuse local`.